plugins/golang/commands/lint-fix.md

```diff
@@ -0,0 +1,105 @@
+---
+description: Run golangci-lint tool and fix all reported issues
+---
+
+## Name
+golang:lint-fix
+
+## Synopsis
+```
+/golang:lint-fix
+/golang:lint-fix [<linter-flags>]
+```
+
+## Description
+This command runs golangci-lint and systematically fixes all reported issues in the codebase. It creates a todo list to track progress and fixes issues by category until all linter checks pass.
```

Comment:
some (or most) repos have the `hack/go-lint.sh` which might be tailored to each repo and also runs it in a containerized way. You might want to think about just  calling that script if it exists